### PR TITLE
Create a folder with the Project Name in the same directory

### LINKED ISSUE #35: Create a folder with the Project Name in the same directory
Now that the user has inputted the project name, it should then create a directory titled the Project Name

### PR DESCRIPTION
**Pull request by martinisiu**
Creates the base directory when the User enters the project name, error if file already exists

### CODE REVIEW

File: code-base/main.py [hunk 1]
```diff
@@ -18,18 +18,28 @@ def titleScreen():
...
 # ------------------------------------------------------------------
 
-
+def createDirectory(projectName):
+    """
+    Creates the root directory to store the web app
+    """
+    if not os.path.exists(projectName):
+        os.makedirs(projectName)
+        return True
```
Comment (ZenaMel) [inline code-base/main.py:39]: Why are you returning a boolean here?

File: code-base/main.py [hunk 2]
```diff
@@ -18,18 +18,28 @@ def titleScreen():
...
+    else:
+        print("[ERROR] A project already exists with the name  '" + projectName + "' ")
+        return False
```
Comment (ZenaMel) [inline code-base/main.py:42]: Why are you returning a boolean here?

File: code-base/main.py [hunk 3]
```diff
@@ -18,18 +18,28 @@ def titleScreen():
...
-
+def createDirectory(projectName):
+    """
+    Creates the root directory to store the web app
+    """
+    if not os.path.exists(projectName):
+        os.makedirs(projectName)
+        return True
+    else:
+        print("[ERROR] A project already exists with the name  '" + projectName + "' ")
```
Comment (ZenaMel) [inline code-base/main.py:41]: This works for now, but we'll introduce proper error-handling down the road.

**Review by ZenaMel — CHANGES_REQUESTED** [review]
Minor changes ought to be made.

File: code-base/main.py (hunk 3 above)
Comment (martinisiu) [reply code-base/main.py:41]: Okay sure, how would you suggest to handle the error?

Comment (ZenaMel) [reply code-base/main.py:41]: for examply by using a try/except block i.e.
```
try:
 
        f()
 
    except Exception as e:
 
        print(e)
```

**Pull request merged**